### PR TITLE
Switch Travis to Circleci

### PR DESCRIPTION
**Pull request by fle-bach**

### CODE REVIEW

**Comment by xcomponentadmin** [comment]
Can one of the admins verify this patch?

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+version: 2
+jobs:
+  checkout_code:
+    docker:
+      - image: xcomponent/mono-git:1.3.0
+    steps:
+      - checkout
+      - run: ./build.sh -t All
```
Comment (maurelio1234) [inline .circleci/config.yml]: add a name

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+version: 2
+jobs:
+  checkout_code:
```
Comment (maurelio1234) [inline .circleci/config.yml]: rename this job, it does more than checking out code

**Comment by maurelio1234** [comment]
@anouarhassine what do you think about this PR?

**Comment by anouarhassine** [comment]
Why is not a circle ci build being triggered yet?

**Comment by fle-bach** [comment]
https://circleci.com/gh/xcomponent/ReactiveXComponent.Net/3?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by anouarhassine** [comment]
Okay. @fle-bach , Now that the Circle Ci trigger is fixed, could you please make the changes mentioned by @maurelio1234 ? Because your config now starts a workflow called _xcomponent_runtime_ and it has one job called _checkout_code_. the workflow should be called something like _xcomponent_reactive_api_ and you can rename the job to better describe what it does or to split it into multiple little jobs.

**Comment by fle-bach** [comment]
Replaced by [https://github.com/xcomponent/ReactiveXComponent.Net/pull/91](
https://github.com/xcomponent/ReactiveXComponent.Net/pull/91)

**Pull request closed without merging**